### DISCUSSION TITLE
Issue since Syno Update 7.2-64570

### DISCUSSION THREAD

**Discussion by Hjlfmir** [007revad/Synology_HDD_db#109] [Q&A] [answered]
Hi everybody,

Since my DS2422+ 7.2-64570 update, the script hasn't any effetct : all my 12 disks + 2 ssd are show as "unverified".
I use the last version of the script (3.0.56) and tried 3.1.59, no success :(
I tried to restore, force, reboot, etc ... same issue.

If someone has an idea ?
Thank you for your help.
Vincent

**Comment by 007revad** [comment]
Can you reply with the output from the script.

**Comment by Hjlfmir** [comment]
Sure, here you have.
`Tâche : Update HDD DB
Heure de début : Sun, 02 Jul 2023 12:08:50 GMT
Heure d'arrêt : Sun, 02 Jul 2023 12:08:56 GMT
État actuel : 0 (Normal)
Sortie/erreur standard :
Synology_HDD_db v3.0.56
DS2422+ DSM 7.2-64570-1
Using options:

HDD/SSD models found: 7
HD204UI,1AQ10001
HDWG11A,1AQ10001
1AQ10001
80.00A80
0603
WD101EFAX-68LDBN0,81.00A81
WD40EFRX-68WT0N0,80.00A80
WD60EFRX-68L0BN1,82.00A82
WD60EFRX-68MYMN1,82.00A82
WD80EFAX-68KNBN0,81.00A81

M.2 drive models found: 1
Samsung SSD 970 EVO Plus 2TB,2B2QEXM7

M.2 PCIe card models found: 1
E10M20-T1

No Expansion Units found

[0;33mHD204UI [0m already exists in [0;36mds2422+_host_v7.db [0m
[0;33mHD204UI [0m already exists in [0;36mds2413+_host_v7.db [0m
[0;33mHD204UI [0m already exists in [0;36mds2413+_host.db [0m
[0;33mHD204UI [0m already exists in [0;36mds2422+_host_v7.db.new [0m
[0;33mHD204UI [0m already exists in [0;36mds2413+_host_v7.db.new [0m
[0;33mHD204UI [0m already exists in [0;36mds2413+_host.db.new [0m
[0;33mHDWG11A
1AQ10001
80.00A80
0603 [0m already exists in [0;36mds2422+_host_v7.db [0m
[0;33mHDWG11A
1AQ10001
80.00A80
0603 [0m already exists in [0;36mds2413+_host_v7.db [0m
[0;33mHDWG11A
1AQ10001
80.00A80
0603 [0m already exists in [0;36mds2413+_host.db [0m
[0;33mHDWG11A
1AQ10001
80.00A80
0603 [0m already exists in [0;36mds2422+_host_v7.db.new [0m
[0;33mHDWG11A
1AQ10001
80.00A80
0603 [0m already exists in [0;36mds2413+_host_v7.db.new [0m
[0;33mHDWG11A
1AQ10001
80.00A80
0603 [0m already exists in [0;36mds2413+_host.db.new [0m
[0;33mWD101EFAX-68LDBN0 [0m already exists in [0;36mds2422+_host_v7.db [0m
[0;33mWD101EFAX-68LDBN0 [0m already exists in [0;36mds2413+_host_v7.db [0m
[0;33mWD101EFAX-68LDBN0 [0m already exists in [0;36mds2413+_host.db [0m
[0;33mWD101EFAX-68LDBN0 [0m already exists in [0;36mds2422+_host_v7.db.new [0m
[0;33mWD101EFAX-68LDBN0 [0m already exists in [0;36mds2413+_host_v7.db.new [0m
[0;33mWD101EFAX-68LDBN0 [0m already exists in [0;36mds2413+_host.db.new [0m
[0;33mWD40EFRX-68WT0N0 [0m already exists in [0;36mds2422+_host_v7.db [0m
[0;33mWD40EFRX-68WT0N0 [0m already exists in [0;36mds2413+_host_v7.db [0m
[0;33mWD40EFRX-68WT0N0 [0m already exists in [0;36mds2413+_host.db [0m
[0;33mWD40EFRX-68WT0N0 [0m already exists in [0;36mds2422+_host_v7.db.new [0m
[0;33mWD40EFRX-68WT0N0 [0m already exists in [0;36mds2413+_host_v7.db.new [0m
[0;33mWD40EFRX-68WT0N0 [0m already exists in [0;36mds2413+_host.db.new [0m
[0;33mWD60EFRX-68L0BN1 [0m already exists in [0;36mds2422+_host_v7.db [0m
[0;33mWD60EFRX-68L0BN1 [0m already exists in [0;36mds2413+_host_v7.db [0m
[0;33mWD60EFRX-68L0BN1 [0m already exists in [0;36mds2413+_host.db [0m
[0;33mWD60EFRX-68L0BN1 [0m already exists in [0;36mds2422+_host_v7.db.new [0m
[0;33mWD60EFRX-68L0BN1 [0m already exists in [0;36mds2413+_host_v7.db.new [0m
[0;33mWD60EFRX-68L0BN1 [0m already exists in [0;36mds2413+_host.db.new [0m
[0;33mWD60EFRX-68MYMN1 [0m already exists in [0;36mds2422+_host_v7.db [0m
[0;33mWD60EFRX-68MYMN1 [0m already exists in [0;36mds2413+_host_v7.db [0m
[0;33mWD60EFRX-68MYMN1 [0m already exists in [0;36mds2413+_host.db [0m
[0;33mWD60EFRX-68MYMN1 [0m already exists in [0;36mds2422+_host_v7.db.new [0m
[0;33mWD60EFRX-68MYMN1 [0m already exists in [0;36mds2413+_host_v7.db.new [0m
[0;33mWD60EFRX-68MYMN1 [0m already exists in [0;36mds2413+_host.db.new [0m
[0;33mWD80EFAX-68KNBN0 [0m already exists in [0;36mds2422+_host_v7.db [0m
[0;33mWD80EFAX-68KNBN0 [0m already exists in [0;36mds2413+_host_v7.db [0m
[0;33mWD80EFAX-68KNBN0 [0m already exists in [0;36mds2413+_host.db [0m
[0;33mWD80EFAX-68KNBN0 [0m already exists in [0;36mds2422+_host_v7.db.new [0m
[0;33mWD80EFAX-68KNBN0 [0m already exists in [0;36mds2413+_host_v7.db.new [0m
[0;33mWD80EFAX-68KNBN0 [0m already exists in [0;36mds2413+_host.db.new [0m
[0;33mSamsung SSD 970 EVO Plus 2TB [0m already exists in [0;36mds2422+_host_v7.db [0m
[0;33mSamsung SSD 970 EVO Plus 2TB [0m already exists in [0;36mds2413+_host_v7.db [0m
[0;33mSamsung SSD 970 EVO Plus 2TB [0m already exists in [0;36mds2413+_host.db [0m
[0;33mSamsung SSD 970 EVO Plus 2TB [0m already exists in [0;36mds2422+_host_v7.db.new [0m
[0;33mSamsung SSD 970 EVO Plus 2TB [0m already exists in [0;36mds2413+_host_v7.db.new [0m
[0;33mSamsung SSD 970 EVO Plus 2TB [0m already exists in [0;36mds2413+_host.db.new [0m
[0;33mSamsung SSD 970 EVO Plus 2TB [0m already exists in [0;36mds2413+_e10m20-t1_v7.db [0m
[0;33mSamsung SSD 970 EVO Plus 2TB [0m already exists in [0;36mds2422+_e10m20-t1_v7.db [0m
[0;33mSamsung SSD 970 EVO Plus 2TB [0m already exists in [0;36mds2422+_e10m20-t1_v7.db.new [0m


Support disk compatibility already enabled.

Support memory compatibility already enabled.

M.2 volume support already enabled.

Drive db auto updates already enabled.

DSM successfully checked disk compatibility.

You may need to [0;36mreboot the Synology [0m to see the changes.`

**Comment by 007revad** [comment]
The ds2413+host files are probably leftover from when you migrated drives from DS2413+ to the DS2422+ but they shouldn't be a problem, but I'd delete them anyway.

DSM won't allow to create a volume on M.2 drives in the E10M20-T1 (officially it will only let use them as a cache).  If you want to use them as a volume you'll need to use my https://github.com/007revad/Synology_M2_volume script.

I don't know if the Samsung drive's 1AQ10001 firmware version is correct, but the Toshiba drive is showing the same firmware version.
```
HD204UI,1AQ10001
HDWG11A,1AQ10001
```

And then there's the 3 strange lines after the Toshiba drive:
```
1AQ10001   <-- Samsung firmware version
80.00A80   <-- WD firmware version
0603       <-- Seagate firmware version
```

The 0603 looks like a Seagate drive firmware version but I don't see any Seagate drives.

And the script is trying to add this as 1 drive, which may have made a mess of the .db files.
```
HDWG11A
1AQ10001
80.00A80
```

What I'd expect to see is your 7 different drive models listed as:
```
HD204UI,1AQ10001
HDWG11A,0603
WD101EFAX-68LDBN0,81.00A81
WD40EFRX-68WT0N0,80.00A80
WD60EFRX-68L0BN1,82.00A82
WD60EFRX-68MYMN1,82.00A82
WD80EFAX-68KNBN0,81.00A81
```

It's bed time here so I'll get back to you tomorrow with some commands to run to find out what's going on.

**Comment by 007revad** [comment]
Can you reply with the output of the following commands:
```
sudo -i
for d in /sys/block/sd*; do syno_hdd_util --ssd_detect | grep "/dev/$(basename -- "$d")"; done
```

If that returns nothing or an error run these commands:
```
sudo -i
for d in /sys/block/sata*; do syno_hdd_util --ssd_detect | grep "/dev/$(basename -- "$d")"; done
```

**Reply by Hjlfmir** [reply]
Hi, thank you rof your help.
Here the result : 

root@Bayek:~# for d in /sys/block/sd*; do syno_hdd_util --ssd_detect | grep "/dev/$(basename -- "$d")"; done
WD40EFRX-68WT0N0     80.00A80     WD-WMC4E0076488      /dev/sata9 no
WD40EFRX-68WT0N0     80.00A80     WD-WMC4E0059035      /dev/sata8 no
WD60EFRX-68MYMN1     82.00A82     WD-WX11DBBGB926      /dev/sata7 no
WD60EFRX-68L0BN1     82.00A82     WD-WX11D48KPH1K      /dev/sata6 no
WD80EFAX-68KNBN0     81.00A81     VAGELRUL             /dev/sata5 no
WD80EFAX-68KNBN0     81.00A81     VAH4P01L             /dev/sata4 no
WD80EFAX-68KNBN0     81.00A81     VAGYAJUL             /dev/sata3 no
WD101EFAX-68LDBN0    81.00A81     VCG6VGWN             /dev/sata2 no
HD204UI              1AQ10001     S2H7J9GZA00162       /dev/sata12 no
HD204UI              1AQ10001     S2H7J9JB803825       /dev/sata11 no
WD40EFRX-68WT0N0     80.00A80     WD-WMC4E0059126      /dev/sata10 no
HDWG11A              0603         6170A08ZFATG         /dev/sata1 no

**Comment by 007revad** [comment]
Actually all you need to run is:
`sudo -i syno_hdd_util --ssd_detect`

And you can delete the drive's serial numbers if you want. There the 4th column.

**Comment by Hjlfmir** [comment]
> Can you reply with the output of the following commands:
> 
> ```
> sudo -i
> for d in /sys/block/sd*; do syno_hdd_util --ssd_detect | grep "/dev/$(basename -- "$d")"; done
> ```
> 
> If that returns nothing or an error run these commands:
> 
> ```
> sudo -i
> for d in /sys/block/sata*; do syno_hdd_util --ssd_detect | grep "/dev/$(basename -- "$d")"; done
> ```

Here the result : 

root@Bayek:~# sudo -i syno_hdd_util --ssd_detect
Model                Firmware     SN                   Dev        is SSD?
WD40EFRX-68WT0N0     80.00A80     WD-WMC4E0076488      /dev/sata9 no
WD40EFRX-68WT0N0     80.00A80     WD-WMC4E0059035      /dev/sata8 no
WD60EFRX-68MYMN1     82.00A82     WD-WX11DBBGB926      /dev/sata7 no
WD60EFRX-68L0BN1     82.00A82     WD-WX11D48KPH1K      /dev/sata6 no
WD80EFAX-68KNBN0     81.00A81     VAGELRUL             /dev/sata5 no
WD80EFAX-68KNBN0     81.00A81     VAH4P01L             /dev/sata4 no
WD80EFAX-68KNBN0     81.00A81     VAGYAJUL             /dev/sata3 no
WD101EFAX-68LDBN0    81.00A81     VCG6VGWN             /dev/sata2 no
HD204UI              1AQ10001     S2H7J9GZA00162       /dev/sata12 no
HD204UI              1AQ10001     S2H7J9JB803825       /dev/sata11 no
WD40EFRX-68WT0N0     80.00A80     WD-WMC4E0059126      /dev/sata10 no
HDWG11A              0603         6170A08ZFATG         /dev/sata1 no
If this is not right, please kindly report this to us

**Comment by 007revad** [comment]
Can you run the following commands to delete the old ds2413+ files:
```
sudo -i
for f in /var/lib/disk-compatibility/ds2413*; do rm "$f"; done
```
Then run these commands and reply back with the results:
```
sudo -i
for f in /dev/sata*; do cat "$f/device/model"; done
```
```
sudo -i
for f in /dev/sata*; do cat "$f/device/rev"; done
```

**Reply by Hjlfmir** [reply]
Hi, thank you :)
Here the restult : 

root@Bayek:~# for f in /dev/sata*; do cat "$f/device/model"; done
cat: /dev/sata1/device/model: Not a directory
cat: /dev/sata10/device/model: Not a directory
cat: /dev/sata10p1/device/model: Not a directory
cat: /dev/sata10p2/device/model: Not a directory
cat: /dev/sata10p5/device/model: Not a directory
cat: /dev/sata10p6/device/model: Not a directory
cat: /dev/sata10p7/device/model: Not a directory
cat: /dev/sata11/device/model: Not a directory
cat: /dev/sata11p1/device/model: Not a directory
cat: /dev/sata11p2/device/model: Not a directory
cat: /dev/sata11p3/device/model: Not a directory
cat: /dev/sata11p5/device/model: Not a directory
cat: /dev/sata11p6/device/model: Not a directory
cat: /dev/sata12/device/model: Not a directory
cat: /dev/sata12p1/device/model: Not a directory
cat: /dev/sata12p2/device/model: Not a directory
cat: /dev/sata12p3/device/model: Not a directory
cat: /dev/sata12p5/device/model: Not a directory
cat: /dev/sata12p6/device/model: Not a directory
cat: /dev/sata1p1/device/model: Not a directory
cat: /dev/sata1p10/device/model: Not a directory
cat: /dev/sata1p2/device/model: Not a directory
cat: /dev/sata1p5/device/model: Not a directory
cat: /dev/sata1p6/device/model: Not a directory
cat: /dev/sata1p7/device/model: Not a directory
cat: /dev/sata1p8/device/model: Not a directory
cat: /dev/sata1p9/device/model: Not a directory
cat: /dev/sata2/device/model: Not a directory
cat: /dev/sata2p1/device/model: Not a directory
cat: /dev/sata2p10/device/model: Not a directory
cat: /dev/sata2p2/device/model: Not a directory
cat: /dev/sata2p5/device/model: Not a directory
cat: /dev/sata2p6/device/model: Not a directory
cat: /dev/sata2p7/device/model: Not a directory
cat: /dev/sata2p8/device/model: Not a directory
cat: /dev/sata2p9/device/model: Not a directory
cat: /dev/sata3/device/model: Not a directory
cat: /dev/sata3p1/device/model: Not a directory
cat: /dev/sata3p2/device/model: Not a directory
cat: /dev/sata3p5/device/model: Not a directory
cat: /dev/sata3p6/device/model: Not a directory
cat: /dev/sata3p7/device/model: Not a directory
cat: /dev/sata3p8/device/model: Not a directory
cat: /dev/sata3p9/device/model: Not a directory
cat: /dev/sata4/device/model: Not a directory
cat: /dev/sata4p1/device/model: Not a directory
cat: /dev/sata4p2/device/model: Not a directory
cat: /dev/sata4p5/device/model: Not a directory
cat: /dev/sata4p6/device/model: Not a directory
cat: /dev/sata4p7/device/model: Not a directory
cat: /dev/sata4p8/device/model: Not a directory
cat: /dev/sata4p9/device/model: Not a directory
cat: /dev/sata5/device/model: Not a directory
cat: /dev/sata5p1/device/model: Not a directory
cat: /dev/sata5p2/device/model: Not a directory
cat: /dev/sata5p5/device/model: Not a directory
cat: /dev/sata5p6/device/model: Not a directory
cat: /dev/sata5p7/device/model: Not a directory
cat: /dev/sata5p8/device/model: Not a directory
cat: /dev/sata5p9/device/model: Not a directory
cat: /dev/sata6/device/model: Not a directory
cat: /dev/sata6p1/device/model: Not a directory
cat: /dev/sata6p2/device/model: Not a directory
cat: /dev/sata6p5/device/model: Not a directory
cat: /dev/sata6p6/device/model: Not a directory
cat: /dev/sata6p7/device/model: Not a directory
cat: /dev/sata6p8/device/model: Not a directory
cat: /dev/sata7/device/model: Not a directory
cat: /dev/sata7p1/device/model: Not a directory
cat: /dev/sata7p2/device/model: Not a directory
cat: /dev/sata7p5/device/model: Not a directory
cat: /dev/sata7p6/device/model: Not a directory
cat: /dev/sata7p7/device/model: Not a directory
cat: /dev/sata7p8/device/model: Not a directory
cat: /dev/sata8/device/model: Not a directory
cat: /dev/sata8p1/device/model: Not a directory
cat: /dev/sata8p2/device/model: Not a directory
cat: /dev/sata8p5/device/model: Not a directory
cat: /dev/sata8p6/device/model: Not a directory
cat: /dev/sata8p7/device/model: Not a directory
cat: /dev/sata9/device/model: Not a directory
cat: /dev/sata9p1/device/model: Not a directory
cat: /dev/sata9p2/device/model: Not a directory
cat: /dev/sata9p5/device/model: Not a directory
cat: /dev/sata9p6/device/model: Not a directory
cat: /dev/sata9p7/device/model: Not a directory
root@Bayek:~# for f in /dev/sata*; do cat "$f/device/rev"; done
cat: /dev/sata1/device/rev: Not a directory
cat: /dev/sata10/device/rev: Not a directory
cat: /dev/sata10p1/device/rev: Not a directory
cat: /dev/sata10p2/device/rev: Not a directory
cat: /dev/sata10p5/device/rev: Not a directory
cat: /dev/sata10p6/device/rev: Not a directory
cat: /dev/sata10p7/device/rev: Not a directory
cat: /dev/sata11/device/rev: Not a directory
cat: /dev/sata11p1/device/rev: Not a directory
cat: /dev/sata11p2/device/rev: Not a directory
cat: /dev/sata11p3/device/rev: Not a directory
cat: /dev/sata11p5/device/rev: Not a directory
cat: /dev/sata11p6/device/rev: Not a directory
cat: /dev/sata12/device/rev: Not a directory
cat: /dev/sata12p1/device/rev: Not a directory
cat: /dev/sata12p2/device/rev: Not a directory
cat: /dev/sata12p3/device/rev: Not a directory
cat: /dev/sata12p5/device/rev: Not a directory
cat: /dev/sata12p6/device/rev: Not a directory
cat: /dev/sata1p1/device/rev: Not a directory
cat: /dev/sata1p10/device/rev: Not a directory
cat: /dev/sata1p2/device/rev: Not a directory
cat: /dev/sata1p5/device/rev: Not a directory
cat: /dev/sata1p6/device/rev: Not a directory
cat: /dev/sata1p7/device/rev: Not a directory
cat: /dev/sata1p8/device/rev: Not a directory
cat: /dev/sata1p9/device/rev: Not a directory
cat: /dev/sata2/device/rev: Not a directory
cat: /dev/sata2p1/device/rev: Not a directory
cat: /dev/sata2p10/device/rev: Not a directory
cat: /dev/sata2p2/device/rev: Not a directory
cat: /dev/sata2p5/device/rev: Not a directory
cat: /dev/sata2p6/device/rev: Not a directory
cat: /dev/sata2p7/device/rev: Not a directory
cat: /dev/sata2p8/device/rev: Not a directory
cat: /dev/sata2p9/device/rev: Not a directory
cat: /dev/sata3/device/rev: Not a directory
cat: /dev/sata3p1/device/rev: Not a directory
cat: /dev/sata3p2/device/rev: Not a directory
cat: /dev/sata3p5/device/rev: Not a directory
cat: /dev/sata3p6/device/rev: Not a directory
cat: /dev/sata3p7/device/rev: Not a directory
cat: /dev/sata3p8/device/rev: Not a directory
cat: /dev/sata3p9/device/rev: Not a directory
cat: /dev/sata4/device/rev: Not a directory
cat: /dev/sata4p1/device/rev: Not a directory
cat: /dev/sata4p2/device/rev: Not a directory
cat: /dev/sata4p5/device/rev: Not a directory
cat: /dev/sata4p6/device/rev: Not a directory
cat: /dev/sata4p7/device/rev: Not a directory
cat: /dev/sata4p8/device/rev: Not a directory
cat: /dev/sata4p9/device/rev: Not a directory
cat: /dev/sata5/device/rev: Not a directory
cat: /dev/sata5p1/device/rev: Not a directory
cat: /dev/sata5p2/device/rev: Not a directory
cat: /dev/sata5p5/device/rev: Not a directory
cat: /dev/sata5p6/device/rev: Not a directory
cat: /dev/sata5p7/device/rev: Not a directory
cat: /dev/sata5p8/device/rev: Not a directory
cat: /dev/sata5p9/device/rev: Not a directory
cat: /dev/sata6/device/rev: Not a directory
cat: /dev/sata6p1/device/rev: Not a directory
cat: /dev/sata6p2/device/rev: Not a directory
cat: /dev/sata6p5/device/rev: Not a directory
cat: /dev/sata6p6/device/rev: Not a directory
cat: /dev/sata6p7/device/rev: Not a directory
cat: /dev/sata6p8/device/rev: Not a directory
cat: /dev/sata7/device/rev: Not a directory
cat: /dev/sata7p1/device/rev: Not a directory
cat: /dev/sata7p2/device/rev: Not a directory
cat: /dev/sata7p5/device/rev: Not a directory
cat: /dev/sata7p6/device/rev: Not a directory
cat: /dev/sata7p7/device/rev: Not a directory
cat: /dev/sata7p8/device/rev: Not a directory
cat: /dev/sata8/device/rev: Not a directory
cat: /dev/sata8p1/device/rev: Not a directory
cat: /dev/sata8p2/device/rev: Not a directory
cat: /dev/sata8p5/device/rev: Not a directory
cat: /dev/sata8p6/device/rev: Not a directory
cat: /dev/sata8p7/device/rev: Not a directory
cat: /dev/sata9/device/rev: Not a directory
cat: /dev/sata9p1/device/rev: Not a directory
cat: /dev/sata9p2/device/rev: Not a directory
cat: /dev/sata9p5/device/rev: Not a directory
cat: /dev/sata9p6/device/rev: Not a directory
cat: /dev/sata9p7/device/rev: Not a directory

**Comment by 007revad** [comment]
Can you download this file and unzip it:
https://github.com/007revad/Synology_HDD_db/raw/test/drive_info2.zip

Then run drive_info2.sh and reply back with the results.

**Reply by Hjlfmir** [reply]
Here you have : 

/sys/block/sata1
Model: 'HDWG11A'
FwRev: '1AQ10001
1AQ10001
80.00A80
0603'

/sys/block/sata10
Model: 'WD40EFRX-68WT0N0'
FwRev: '80.00A80'

/sys/block/sata11
Model: 'HD204UI'
FwRev: '1AQ10001'

/sys/block/sata12
Model: 'HD204UI'
FwRev: '1AQ10001'

/sys/block/sata2
Model: 'WD101EFAX-68LDBN0'
FwRev: '81.00A81'

/sys/block/sata3
Model: 'WD80EFAX-68KNBN0'
FwRev: '81.00A81'

/sys/block/sata4
Model: 'WD80EFAX-68KNBN0'
FwRev: '81.00A81'

/sys/block/sata5
Model: 'WD80EFAX-68KNBN0'
FwRev: '81.00A81'

/sys/block/sata6
Model: 'WD60EFRX-68L0BN1'
FwRev: '82.00A82'

/sys/block/sata7
Model: 'WD60EFRX-68MYMN1'
FwRev: '82.00A82'

/sys/block/sata8
Model: 'WD40EFRX-68WT0N0'
FwRev: '80.00A80'

/sys/block/sata9
Model: 'WD40EFRX-68WT0N0'
FwRev: '80.00A80'

Before sorting:
HDWG11A,1AQ10001
1AQ10001
80.00A80
0603
WD40EFRX-68WT0N0,80.00A80
HD204UI,1AQ10001
HD204UI,1AQ10001
WD101EFAX-68LDBN0,81.00A81
WD80EFAX-68KNBN0,81.00A81
WD80EFAX-68KNBN0,81.00A81
WD80EFAX-68KNBN0,81.00A81
WD60EFRX-68L0BN1,82.00A82
WD60EFRX-68MYMN1,82.00A82
WD40EFRX-68WT0N0,80.00A80
WD40EFRX-68WT0N0,80.00A80

HDD/SSD models found: 7
HD204UI,1AQ10001
HDWG11A,1AQ10001
1AQ10001
80.00A80
0603
WD101EFAX-68LDBN0,81.00A81
WD40EFRX-68WT0N0,80.00A80
WD60EFRX-68L0BN1,82.00A82
WD60EFRX-68MYMN1,82.00A82
WD80EFAX-68KNBN0,81.00A81

**Comment by 007revad** [comment]
I see the problem. For sata1 it's getting the firmware version sata11, sata12, sata10 and sata1

Does this command return only 0603 ?
`sudo syno_hdd_util --ssd_detect | grep "sata1 " | awk '{print $(NF-3)}'`

**Reply by Hjlfmir** [reply]
Yes, only 0603 :)

**Comment by 007revad** [comment]
Perfect. Can you try this updated version of the script:
https://github.com/007revad/Synology_HDD_db/archive/refs/tags/v3.1.60-RC.zip

**Reply by Hjlfmir** [reply]
Thanks a lot
I did.
Here the result : 

root@Bayek:~# /volume1/homes/Vincent/Drive/Downloads/NAS/Synology_HDD_db/syno_hdd_db.sh
Synology_HDD_db v3.1.60
DS2422+ DSM 7.2-64570-1
Using options:

HDD/SSD models found: 7
HD204UI,1AQ10001
HDWG11A,0603
WD101EFAX-68LDBN0,81.00A81
WD40EFRX-68WT0N0,80.00A80
WD60EFRX-68L0BN1,82.00A82
WD60EFRX-68MYMN1,82.00A82
WD80EFAX-68KNBN0,81.00A81

M.2 drive models found: 1
Samsung SSD 970 EVO Plus 2TB,2B2QEXM7

M.2 PCIe card models found: 1
E10M20-T1

No Expansion Units found

HD204UI already exists in ds2422+_host_v7.db
HD204UI already exists in ds2422+_host_v7.db.new
HDWG11A already exists in ds2422+_host_v7.db
Added HDWG11A to ds2422+_host_v7.db.new
WD101EFAX-68LDBN0 already exists in ds2422+_host_v7.db
WD101EFAX-68LDBN0 already exists in ds2422+_host_v7.db.new
WD40EFRX-68WT0N0 already exists in ds2422+_host_v7.db
WD40EFRX-68WT0N0 already exists in ds2422+_host_v7.db.new
WD60EFRX-68L0BN1 already exists in ds2422+_host_v7.db
WD60EFRX-68L0BN1 already exists in ds2422+_host_v7.db.new
WD60EFRX-68MYMN1 already exists in ds2422+_host_v7.db
WD60EFRX-68MYMN1 already exists in ds2422+_host_v7.db.new
WD80EFAX-68KNBN0 already exists in ds2422+_host_v7.db
WD80EFAX-68KNBN0 already exists in ds2422+_host_v7.db.new
Samsung SSD 970 EVO Plus 2TB already exists in ds2422+_host_v7.db
Samsung SSD 970 EVO Plus 2TB already exists in ds2422+_host_v7.db.new
Samsung SSD 970 EVO Plus 2TB already exists in ds2422+_e10m20-t1_v7.db
Samsung SSD 970 EVO Plus 2TB already exists in ds2422+_e10m20-t1_v7.db.new

E10M20-T1 NIC already enabled for DS2422+
E10M20-T1 NVMe already enabled for DS2422+
E10M20-T1 SATA already enabled for DS2422+
E10M20-T1 already enabled in model.dtb

Support disk compatibility already enabled.

Support memory compatibility already enabled.

M.2 volume support already enabled.

Drive db auto updates already enabled.

DSM successfully checked disk compatibility.

But after that, even after a reboot, same issue : all disks are "unverified".
(I tried to make a --restore before, same thing).

**Comment by 007revad** [comment]
Try running it with the -nf options:
`/volume1/homes/Vincent/Drive/Downloads/NAS/Synology_HDD_db/syno_hdd_db.sh -nf`

Then reboot.

**Reply by Hjlfmir** [reply]
I dit it, restart, but no changes :(

**Comment by 007revad** [comment]
Can you replace the **ds2422+_host_v7.db** and **ds2422+_e10m20-t1_v7.db** files in **/var/lib/disk-compatibility** with ones in the attached zip file.

Then run the script again with **-nf** and reboot.

**Reply by Hjlfmir** [reply]
THank you, but I can't see any zip file joined :/

**Comment by 007revad** [comment]
Oops. I've attached the zip file this time.
[disk-compatibility.zip](https://github.com/007revad/Synology_HDD_db/files/11935609/disk-compatibility.zip)

**Reply by Hjlfmir** [reply]
Thanks :)

Done. I started by removed old files before copy those you provided.
Run script with -nf, restarted, but still the same :(

root@Bayek:/var/lib/disk-compatibility# /volume1/homes/Vincent/Drive/Downloads/NAS/Synology_HDD_db/syno_hdd_db.sh -nf
Synology_HDD_db v3.1.60
DS2422+ DSM 7.2-64570-1
Using options: -nf

HDD/SSD models found: 7
HD204UI,1AQ10001
HDWG11A,0603
WD101EFAX-68LDBN0,81.00A81
WD40EFRX-68WT0N0,80.00A80
WD60EFRX-68L0BN1,82.00A82
WD60EFRX-68MYMN1,82.00A82
WD80EFAX-68KNBN0,81.00A81

M.2 drive models found: 1
Samsung SSD 970 EVO Plus 2TB,2B2QEXM7

M.2 PCIe card models found: 1
E10M20-T1

No Expansion Units found

Added HD204UI to ds2422+_host_v7.db
HD204UI already exists in ds2422+_host_v7.db.new
Added HDWG11A to ds2422+_host_v7.db
HDWG11A already exists in ds2422+_host_v7.db.new
Added WD101EFAX-68LDBN0 to ds2422+_host_v7.db
WD101EFAX-68LDBN0 already exists in ds2422+_host_v7.db.new
Added WD40EFRX-68WT0N0 to ds2422+_host_v7.db
WD40EFRX-68WT0N0 already exists in ds2422+_host_v7.db.new
Added WD60EFRX-68L0BN1 to ds2422+_host_v7.db
WD60EFRX-68L0BN1 already exists in ds2422+_host_v7.db.new
Added WD60EFRX-68MYMN1 to ds2422+_host_v7.db
WD60EFRX-68MYMN1 already exists in ds2422+_host_v7.db.new
Added WD80EFAX-68KNBN0 to ds2422+_host_v7.db
WD80EFAX-68KNBN0 already exists in ds2422+_host_v7.db.new
Added Samsung SSD 970 EVO Plus 2TB to ds2422+_host_v7.db
Samsung SSD 970 EVO Plus 2TB already exists in ds2422+_host_v7.db.new
Added Samsung SSD 970 EVO Plus 2TB to ds2422+_e10m20-t1_v7.db
Samsung SSD 970 EVO Plus 2TB already exists in ds2422+_e10m20-t1_v7.db.new

E10M20-T1 NIC already enabled for DS2422+
E10M20-T1 NVMe already enabled for DS2422+
E10M20-T1 SATA already enabled for DS2422+
E10M20-T1 already enabled in model.dtb

Disabled support disk compatibility.

Support memory compatibility already enabled.

M.2 volume support already enabled.

Disabled drive db auto updates.

DSM successfully checked disk compatibility.

You may need to reboot the Synology to see the changes.

**Reply by Hjlfmir** [reply]
The weird thing is, before to start working, script was working well and I had just an issue with Disk #1.
But after script runs, nothing at all was working :/

**Comment by 007revad** [comment]
It is weird that you were only getting a warning on the Toshiba drive. I'm starting to think with DSM 7.2 the xs+ models may also need the db files in /var.defaults/lib edited as well.

Can you try the following script. It makes DSM think all your drives are Synology drives.
https://github.com/007revad/Synology_enable_M2_volume

**Reply by Hjlfmir** [reply]
Ok, so I did it.
Scrip ran well, I rebooted, but ... still the same issue :'(

**Comment by 007revad** [comment]
I've edited by hand the ds2422+_host_v7.db file and attached it to this reply.
[ds2422+_host_v7.zip](https://github.com/007revad/Synology_HDD_db/files/11949601/ds2422%2B_host_v7.zip)

Can you unzip it and copy the new **`ds2422+_host_v7.db`** file to **`/var/lib/disk-compatibility`** then reboot and check storage manager.

If that doesn't work then also copy the new **`ds2422+_host_v7.db`** file to **`/var.defaults/lib/disk-compatibility`** then reboot and check storage manager.

**Reply by Hjlfmir** [reply]
Hi, thank you for your help.
I copied the file into the both locations, but still does not work :(
I realy wonder if there isn't something else as when it strated to "not work" only disk1 was preseting issue. After applying one more time the script, all gone down :( and then no way to get it work

**Comment by 007revad** [comment, marked as answer]
I think things got messed up when the script added drive 1 as:
```
HDWG11A
1AQ10001
80.00A80
0603
```
Instead of as HDWG11A with firmware version 0630

Are you willing to reinstall the same DSM version? That will restore all the changed files, without losing your data or settings. 

https://github.com/007revad/Synology_DSM_reinstall/blob/main/README.md

**Reply by Hjlfmir** [reply]
Hey ! Amazing ! A lot better ! :D
Only disk 2 (WD101EFAX) is shown as "unverified".
After runing the script, all disks are well verified !

Thank you very much for your support :)
